sqlitecluster/SQLiteNode.cpp

```diff
@@ -1645,7 +1648,8 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
                 auto threadID = _replicationThreadCount.fetch_add(1);
                 SDEBUG("Spawning concurrent replicate thread (blocks until DB handle available): " << threadID);
                 try {
-                    thread(&SQLiteNode::_replicate, this, peer, message, _dbPool->getIndex(false)).detach();
+                    uint64_t threadAttemptStartTimestamp = STimeNow();
+                    thread(&SQLiteNode::_replicate, this, peer, message, _dbPool->getIndex(false), threadAttemptStartTimestamp).detach();
```

Comment:
is there a reason to create the variable instead of just using `STimeNow()` directly here?
```suggestion
                    thread(&SQLiteNode::_replicate, this, peer, message, _dbPool->getIndex(false), STimeNow()).detach();
```